source/conf.py

```diff
@@ -115,20 +115,20 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #html_theme = 'sphinxdoc'
-html_theme = 'bootstrap'
-html_theme_path = sphinx_bootstrap_theme.get_html_theme_path()
+html_theme = 'pydy'
+html_theme_path = ['../_theme']
```

Comment:
Why do we need to include _theme in source directory, will it not cause some unwanted pages when autogenerated?